src/CommunityToolkit.Maui/Views/RatingView/RatingView.Shared.cs

```diff
@@ -0,0 +1,361 @@
+
+
+	/// <summary>Star Color Bindable property.</summary>
+	public static readonly BindableProperty FilledBackgroundColorProperty = BindableProperty.Create(nameof(FilledBackgroundColor), typeof(Color), typeof(RatingView), defaultValue: Colors.Yellow, propertyChanged: OnBindablePropertyChanged);
+
+	///<summary></summary>
+	public static readonly BindableProperty EmptyBackgroundColorProperty = BindableProperty.Create(nameof(EmptyBackgroundColor), typeof(Color), typeof(RatingView), defaultValue: Colors.White, propertyChanged: OnBindablePropertyChanged);
+
+	///<summary></summary>
+	public static readonly BindableProperty StrokeColorProperty = BindableProperty.Create(nameof(StrokeColor), typeof(Color), typeof(RatingView), defaultValue: Colors.Gray, propertyChanged: OnBindablePropertyChanged);
+
+
+	///<summary></summary>
+	public static readonly BindableProperty SpacingProperty = BindableProperty.Create(nameof(Spacing), typeof(double), typeof(RatingView), defaultValue: 10.0, propertyChanged: OnBindablePropertyChanged);
+
+	///<summary></summary>
+	public static readonly BindableProperty ShouldAllowRatingProperty = BindableProperty.Create(nameof(ShouldAllowRating), typeof(bool), typeof(RatingView), defaultValue: false, propertyChanged: OnBindablePropertyChanged);
+
+	///<summary></summary>
+	public static readonly BindableProperty CommandProperty = BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(RatingView), defaultValue: null, propertyChanged: OnBindablePropertyChanged);
+
+	///<summary></summary>
+	public static readonly BindableProperty CommandParameterProperty = BindableProperty.Create(nameof(CommandParameter), typeof(object), typeof(RatingView), defaultValue: null, propertyChanged: OnBindablePropertyChanged);
+
+	///<summary></summary>
+	public readonly BindableProperty ShapeProperty = BindableProperty.Create(nameof(Shape), typeof(RatingShape), typeof(RatingView), propertyChanged: OnShapePropertyChanged);
+
+
+	///<summary></summary>
+	public double CurrentRating
+	{
+		get => (double)GetValue(CurrentRatingProperty);
+		set => SetValue(CurrentRatingProperty, value);
+	}
+
+	///<summary></summary>
+	public int MaximumRating
+	{
+		get => (int)GetValue(MaximumRatingProperty);
+		set => SetValue(MaximumRatingProperty, value);
+	}
+
+	///<summary></summary>
+	public double Size
+	{
+		get => (double)GetValue(SizeProperty);
+		set => SetValue(SizeProperty, value);
+	}
+
+	///<summary></summary>
+	public Color FilledBackgroundColor
+	{
+		get => (Color)GetValue(FilledBackgroundColorProperty);
+		set => SetValue(FilledBackgroundColorProperty, value);
+	}
+
+	///<summary></summary>
+	public Color EmptyBackgroundColor
+	{
+		get => (Color)GetValue(EmptyBackgroundColorProperty);
+		set => SetValue(EmptyBackgroundColorProperty, value);
+	}
+
+	///<summary></summary>
+	public Color StrokeColor
```

Comment:
Border should already have such properties. There can be a confusion which one to use